Source/WebCore/Modules/mediasource/MediaSource.cpp

```diff
@@ -978,6 +1004,9 @@ void MediaSource::detachFromElement(HTMLMediaElement& element)
 
     m_private = nullptr;
     m_mediaElement = nullptr;
+
+    if (m_seekCompletedHandler)
+        m_seekCompletedHandler(MediaTime::invalidTime());
```

Comment:
Is it the only case where we need to call m_seekCompletedHandler?
What about when it gets stopped for instance?

Reply:
done